daemons/gptp/common/ptp_message.cpp

```diff
@@ -1502,12 +1510,23 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	    Timestamp prev_peer_ts_theirs;
 	    FrequencyRatio rate_offset;
 	    if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+		    FrequencyRatio upper_limit, lower_limit;
+		    upper_limit = UPPER_FREQ_LIMIT*2;
+		    upper_limit /= US_PER_SEC;
+		    upper_limit += 1;
+		    lower_limit = UPPER_FREQ_LIMIT*2;
```

Comment:
What is the advantage of all the complicated math?
Couldn't we just go
upper_limit_ppm = 1.000250  // Upper limit of the neighbor rate ratio in PPM
lower_limit_ppm = 0.999750  // Lower limit of the neighbor rate ratio in PPM